src/components/Search/SearchListWithHeader.tsx

```diff
@@ -53,6 +54,8 @@ function SearchListWithHeader(
     const [selectedItems, setSelectedItems] = useState<SelectedTransactions>({});
     const [selectedItemsToDelete, setSelectedItemsToDelete] = useState<string[]>([]);
     const [deleteExpensesConfirmModalVisible, setDeleteExpensesConfirmModalVisible] = useState(false);
+    const [offlineModalVisible, setOfflineModalVisible] = useState(false);
+    const [selectedReports, setSelectedReports] = useState<string[]>([]);
```

Comment:
do you mind renaming `selectedItems` to `selectedTransactions` so that we are aligned with the naming.
Also move these two use states to be next to each other, since they are handling very similar data.

Reply:
```suggestion
    const [selectedReports, setSelectedReports] = useState<SearchReport["reportID"][]>([]);
```

seems cleaner

Reply:
Applied